scio-test/src/test/scala/com/spotify/scio/ScioContextTest.scala

```diff
@@ -175,4 +175,42 @@ class ScioContextTest extends PipelineSpec {
     the[IllegalArgumentException] thrownBy { ScioContext.apply(invalidOpts) } should have message
       s"blockFor param foo cannot be cast to type scala.concurrent.duration.Duration"
   }
+
```

Comment:
are these test cases needed? I saw that the part was already covered in codecov, but couldn't find where the test case was !!

Reply:
I don't think we need them. If you replace the internal call with the new one.